magellan-legacy/src/main/java/com/wealthfront/magellan/LegacyStep.kt

```diff
@@ -0,0 +1,23 @@
+package com.wealthfront.magellan
+
+import android.content.Context
+import android.view.View
+import androidx.annotation.VisibleForTesting
+import com.wealthfront.magellan.core.Navigable
+import com.wealthfront.magellan.lifecycle.LifecycleAwareComponent
+import com.wealthfront.magellan.lifecycle.attachFieldToLifecycle
+import com.wealthfront.magellan.lifecycle.createAndAttachFieldToLifecycleWhenShown
+import com.wealthfront.magellan.view.DialogComponent
+
+public abstract class LegacyStep<V : View>(protected val navigator: Navigator) :
```

Comment:
I am hoping that asking for a navigator argument will force developers to think about / maintain the backstack "scope" at which this LegacyStep operates